regression/cbmc/void_pointer3/main.c

```diff
@@ -0,0 +1,12 @@
+int main()
+{
+  // make sure they are NULL objects
+  void *p=0, *q=0;
+  unsigned o1, o2;
```

Comment:
I would prefer uintNN_t's here so that we don't have to worry about architectural differences.